routes/mountains.js

```diff
@@ -0,0 +1,191 @@
+</ul>
+</div>
+`;
+
+app.get('/', async(req, res, next) => {
+  try{
+    const ranges = await db.models.Range.findAll();
+
+    let html = `
+    <!DOCTYPE html>
+    <html lang="en">
+      ${head}
+      <body>
+        ${navBar}
+        <div id="main-content">
+        <h1>The Fourtneeners</h1>
+        <h2 style="font-size: .8rem">Sequelize & PostgreSQL Version</h2>
+        <p class="description"> <strong>Fourteener</strong> fȯr-ˈtēn-ər<br /> A mountain peek with an elevation of at least 14,000 ft (4267 m).</p>
+          <div id="entry-list">
+            ${ranges.map( range => `
+              <div class="entryContainer">
+                <h2>${range["Name"]}</h2>
+                <div id="button-${range.ID}" class="details-container"><a href="/mountains/${range.ID}">See details</a></div>
+              </div>
+            `).join("")}        
+            
+          </div>
+        </div>
+      </body>
+    </html>
+    `
+    res.send(`
+    ${html}
+    `
+    );
+  }
+  catch(error) {
+    next(error);
+  }
+})
+
+app.get('/:range', async(req, res, next) => {
+  try {
+    const range = await db.models.Mountain.findAll(
+      {
+        where : {
+          RangeID: req.params.range
+        }
+      }
+    )
+    let randomIdx = () => Math.floor(Math.random() * range.length);
+    let sampleMountain = range[randomIdx()];
+    
+    const hero = `
+    <div id="hero" style="background-image: url(${sampleMountain.photo})">
+    <h1 class="h3">Range: ${sampleMountain["Mountain Range"].toUpperCase()}</h1>
+    </div>
+    `;
+
+    res.send(`
+      <!DOCTYPE html>
+      <html lang="en">
+        ${head}
+        <body id="indiv-body">
+          ${navBar}
+          ${hero}
+            <div id="entry-list">              
+              ${range.map( mountain => `
+                <div class="entryContainer ${mountain.Difficulty} ${mountain["Mountain Range"]}" id="range${mountain.ID}" difficulty="${mountain.Difficulty}" mountainRange="${mountain['Mountain Range']}">
+                  <h2>${mountain["Mountain Peak"]}</h2>
+                  <p>Range: ${mountain["Mountain Range"]}</p>
+                  <p>Elevation: ${mountain.Elevation_ft}</p>
+                  <div id="button-${mountain.ID}" class="details-container"><a href="/mountains/${mountain.RangeID}/${mountain.ID}">See details</a></div>
+                </div>
+              `).join("")}
+            </div>
+        </body>
+      </html>
+    `);
+  }
+  catch (error) {
+    next(error);
+  }
+})
+
+app.get('/:range/:id', async(req, res, next) => {
+  try {
+    const mountain = await db.models.Mountain.findOne(
+      {
+        where : {
+          ID: req.params.id
+        }
+      }
+    )
+    
+    // Comment for Stanley: I had to recreate the below 'head' and 'navBar' variables just to change the CSS stylesheet href to ".././style.css". 
+    // Is there a better way to setup my filepaths? I tried a few methods but couldn't get them to work! 
```

Comment:
i think it's the best way rn- filepaths with css files and html files are super annoying and inconsistent idk why